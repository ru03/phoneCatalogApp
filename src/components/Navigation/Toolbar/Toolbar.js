import React from 'react';
import classes from './Toolbar.module.css'
const toolbar = (props) => (
    <header className={classes.Toolbar}>
        <div>Phone Catalog</div>
    </header>
)

export default toolbar;