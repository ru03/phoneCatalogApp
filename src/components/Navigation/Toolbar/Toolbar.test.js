import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Toolbar from './Toolbar';

describe('Toolbar test', () => {
    it('Toolbar component should be displayed', () => {
        const toolbar = shallow(<Toolbar></Toolbar>);

        expect(toolbar).toMatchSnapshot();
    })
})