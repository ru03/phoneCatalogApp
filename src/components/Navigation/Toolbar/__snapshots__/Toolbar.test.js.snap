// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar test Toolbar component should be displayed 1`] = `
<header
  className="Toolbar"
>
  <div>
    Phone Catalog
  </div>
</header>
`;
